openmls/src/extensions/external_pub_extension.rs

```diff
@@ -0,0 +1,29 @@
+use tls_codec::{TlsDeserialize, TlsSerialize, TlsSize};
+
+use super::{Deserialize, Serialize};
+use crate::prelude::HpkePublicKey;
+
+/// ```c
+/// // draft-ietf-mls-protocol-16
+/// struct {
+///     HPKEPublicKey external_pub;
+/// } ExternalPub;
+/// ```
+#[derive(
+    PartialEq, Eq, Clone, Debug, Serialize, Deserialize, TlsSerialize, TlsDeserialize, TlsSize,
+)]
+pub struct ExternalPubExtension {
```

Comment:
Wasn't there a test for this extension? Would be nice to have at least something along the lines of the others in `test_extensions`.

Reply:
No there wasn't. But I will add something :+1:

Reply:
Ah, I think you thought about this one https://github.com/openmls/openmls/pull/1085/files#diff-7f947c709686d3816215e94503d38aafeb8990e9a8b60230fedbbc1aed50bfd8R31 I was also confused for a moment :-)